pages/docs/couple-your-code/couple-your-code-porting-v2-3.md

```diff
@@ -201,58 +208,67 @@ error: ‘class precice::SolverInterface’ has no member named ‘initializeDat
+  - Replace `<export:vtk />` for parallel participants with `<export:vtu />` or `<export:vtp />`.
+  - Replace mapping constraint `scaled-consistent` with `scaled-consistent-surface`.
+  - Remove all timings in the mapping configuration `<mapping: ... timing="initial/onadvance/ondemand" />`.
+  - Remove the preallocations in the mapping configuration `<mapping: ... preallocation="tree/compute/estimate/save/off" />`. We always use the superior `tree` method.
+  - Replace all RBF related `<mapping:rbf-... />` tags. RBF mappings are now defined in terms of the applied solver (current options `<mapping:rbf-global-direct ...`, `<mapping:rbf-global-iterative` or `<mapping:rbf-pum-direct ...`) and the applied basis function as a subtag of the solver. Users should use the additionally added auto selection of an appropriate solver, which omits the solver specification, as follows:
+
+      ```xml
+      <mapping:rbf  ...>
+        <basis-function:... />
+      </mapping:rbf>
+      ```
+
+      Example:
+
+      preCICE version 2 rbf configuration:
+
+      ```xml
+      <mapping:compact-polynomial-c0 direction="read" from= ... support-radius="0.3" />
+      ```
+
+      corresponding preCICE version 3 rbf configuration (using the recommended auto selection):
+
+      ```xml
+      <mapping:rbf  direction="read" from= ...>
+        <basis-function:compact-polynomial-c0 support-radius="0.3" />
+      </mapping:rbf>
+      ```
+
+      A specific solver should only be configured if you want to force preCICE to use and stick to a certain solver, independent of your problem size and execution.
+
+  - Rename `<mapping:rbf... use-qr-decomposition="true" />` to `<mapping:rbf-global-direct ... > <basis-function:... /> </mapping:rbf-global-direct>`.
+  - We dropped quite some functionality concerning [data actions](https://precice.org/configuration-action.html) as these were not used to the best of our knowledge and hard to maintain:
+    - Removed deprecated action timings `regular-prior`, `regular-post`, `on-exchange-prior`, and `on-exchange-post`.
+    - Removed action timings `read-mapping-prior`, `write-mapping-prior`, and `on-time-window-complete-post`.
+    - Removed `ComputeCurvatureAction` and `ScaleByDtAction` actions.
+    - Removed callback functions `vertexCallback` and `postAction` from `PythonAction` interface.
+    - Removed timewindowsize from the `performAction` signature of `PythonAction`. The new signature is `performAction(time, data)`
+    - Using actions with multiple couping schemes and mixed time window sizes is not well defined!
+
+- Coupling schemes
+  - Remove `<extraplation-order value="..." />` in `<coupling-scheme>`. Contact us if you need this feature.
+  - Replace `<min-iteration-convergence-measure min-iterations="3" ... />` by `<min-iterations value="3"/>`. No convergence measures won't lead to an error and iterate until `max-iterations`.
```

Comment:
```suggestion
  - Replace `<min-iteration-convergence-measure min-iterations="3" ... />` by `<min-iterations value="3"/>`. No, convergence measures won't lead to an error and iterate until `max-iterations`.
```

Reply:
"No convergence measures" as in 

```suggestion
  - Replace `<min-iteration-convergence-measure min-iterations="3" ... />` by `<min-iterations value="3"/>`. Defining no convergence measures now iterates until `max-iterations`.
```

Reply:
Okay, I would then formulate it in a different way: 

Not defining convergence measures leads to iterations run until `max-iterations` is reached.

Reply:
```suggestion
  - Replace `<min-iteration-convergence-measure min-iterations="3" ... />` by `<min-iterations value="3"/>`. Not defining convergence measures leads to iterations until `max-iterations` is reached.
```